source/locale/en/symbols.dic

```diff
@@ -172,6 +230,9 @@ _	line	most
 ⊇	superset of or equal to	none
 
 # Vulgur Fractions U+2150 to U+215E
+¼	one quarter	none
+½	one half	none
+¾	three quarter	none

```

Comment:
shouldn't this be "three quarters" (with an 's' on the end)

Reply:
"with an s" seems more used indeed